README.md

```diff
@@ -1,53 +1,58 @@
+This extension is developed atop the
+[libmongoc](https://github.com/mongodb/mongo-c-driver) and
+[libbson](https://github.com/mongodb/libbson) libraries. It provides a minimal
+API for core driver functionality: commands, queries, writes, connection
+management, and BSON serialization.
 
-The driver is written to be a bare bone layer to talk to MongoDB, and
-therefore misses many convenience features. Instead, these convenience methods
-have been split out into a layer written in PHP, the
-[MongoDB Library](http://mongodb.github.io/mongo-php-library/).
-Using this library should be your preferred way of interacting with MongoDB.
-
-Please note that the new HHVM and PHP drivers implement a **different API**
-from the legacy driver at http://pecl.php.net/package/mongo; therefore 
-existing libraries that use the legacy driver (e.g. 
-[Doctrine MongoDB's ODM](http://doctrine-mongodb-odm.readthedocs.org/en/latest/))
-will not work with the new drivers.
-
-In the long run, we hope that userland packages will be built atop this driver
-to implement various APIs (e.g. a BC layer for the existing driver, new fluent
-interfaces), management utilities (for creating admin utilities and cluster
-management applications), and other interesting libraries.
+Userland PHP libraries that depend on this extension may provide higher level
+APIs, such as query builders, individual command helper methods, and GridFS.
+Application developers should consider using this extension in conjunction with
+the [MongoDB PHP library](https://github.com/mongodb/mongo-php-library), which
+implements the same higher level APIs found in MongoDB drivers for other
+languages (as well as the
+[legacy PHP driver](https://php.net/manual/en/book.mongo.php)).
 
 ## Documentation
 
-- http://docs.php.net/set.mongodb
+ - https://php.net/manual/en/set.mongodb.php
+ - https://docs.mongodb.com/ecosystem/drivers/php/
 
 ## Installation
 
 To build and install the driver:
 
-```
-$ pecl install mongodb
-$ echo "extension=mongodb.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
-```
+    $ pecl install mongodb
+    $ echo "extension=mongodb.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
+
+Additional installation instructions may be found in the
+[PHP.net documentation](https://php.net/manual/en/mongodb.installation.php).
+
+This extension is intended to be used alongside the
+[MongoDB PHP Library](https://github.com/mongodb/mongo-php-library), which is
+distributed as the
+[`mongodb/mongodb`](https://packagist.org/packages/mongodb/mongodb) package for
+for [Composer](https://getcomposer.org).
+
+## Reporting Issues
 
-We recommend using this extension in conjunction with our
-[userland library](https://github.com/mongodb/mongo-php-library),
-which is distributed as
-[mongodb/mongodb](https://packagist.org/packages/mongodb/mongodb) for Composer.
+Issues may be reported in either
+[GitHub](https://github.com/mongodb/mongo-php-driver/issues/new) or
```

Comment:
I would leave GitHub out of here. We want people to use Jira, but if they file them on GitHub, then we'll deal with them.

Reply:
Alrighty. I might as well rephrase this to suggest people not use either for general support questions and promote https://docs.mongodb.com/manual/support/ while we're at it.